product_docs/docs/pem/8/pem_upgrade/01_upgrading_pem_installation/05_configuring_the_pem_server_on_linux_platforms.mdx

```diff
@@ -11,29 +11,8 @@ After upgrading the PEM Server you can use the following command to configure th
-| `-p`       | The port number of the PEM database server.                                                                                                                                                                                                                             |
-| `-ps`      | The service name of the pemagent; the default value is pemagent.                                                                                                                                                                                                        |
-| `-sp`      | The superuser password of the PEM database server. This value is required.                                                                                                                                                                                              |
-| `-su`      | The superuser name of the PEM database server.                                                                                                                                                                                                                          |
-| `-t`       | The installation type: Specify 1 if the configuration is for web services and backend database, 2 if you are configuring web services, or 3 if you are configuring the backend database. If you specify 3, please note that the database must reside on the local host. |
-| `-un`      | To unregister the PEM server.                                                                                                                                                                                                                                           |
-| `-h`       | Displays help.                                                                                                                                                                                                                                                          |
-
-If you do not provide configuration properties on the command line, you will be prompted for values by the script. To view script-related help, use the command:
-
-```text
-/usr/edb/pem/bin/configure-pem-server.sh --help
-```
+While running the configure script, it will take the values from the old PEM Server configuration file and run the script.
 
 After executing the PEM server configuration file, use your version-specific service control command to restart the `httpd` service.
 
-For detailed information about using an RPM package to install or configure the PEM server or PEM Agent, please see the [PEM Linux Installation Guide](https://www.enterprisedb.com/docs/pem/latest/).
+For detailed information about using an RPM package to install or configure the PEM server or PEM Agent, please see the [PEM Linux Installation Guide](https://www.enterprisedb.com/docs/pem/latest/pem_inst_guide_linux/).
```

Comment:
If you change the names of the install guide, update this (and any other) references to it.

Reply:
I have changed only the the titles inside the index.mdx file so the link references do not need to be updated